experiments/compositions/composition/api/v1alpha1/composition_types.go

```diff
@@ -45,57 +43,67 @@ type ResourceRef struct {
 	// resource: ServiceIdentity.serviceusage.cnrm.cloud.google.com/v1beta1//sqladmin.googleapis.com
 
 	// OPTION 2
-	Group      string `json:"group,omitempty" protobuf:"bytes,1,name=group"`
-	Version    string `json:"version,omitempty" protobuf:"bytes,2,opt,name=version"`
-	Kind       string `json:"kind" protobuf:"bytes,3,name=kind"`
-	Name       string `json:"name,omitempty" protobuf:"bytes,4,name=name"`
-	NameSuffix string `json:"nameSuffix,omitempty" protobuf:"bytes,2,name=nameSuffix"`
+	Group      string `json:"group,omitempty"`
+	Version    string `json:"version,omitempty"`
+	Kind       string `json:"kind"`
+	Name       string `json:"name,omitempty"`
+	NameSuffix string `json:"nameSuffix,omitempty"`
 }
 
 type FieldRef struct {
-	Path string `json:"path" protobuf:"bytes,1,name=path"`
-	As   string `json:"as" protobuf:"bytes,2,name=as"`
+	Path string `json:"path"`
+	As   string `json:"as"`
 }
 
 type ValuesFrom struct {
-	Name        string      `json:"name" protobuf:"bytes,1,name=name"`
-	ResourceRef ResourceRef `json:"resourceRef" protobuf:"resourceRef,2,name=resourceref"`
-	FieldRef    []FieldRef  `json:"fieldRef" protobuf:"fieldRef,3,name=fieldref"`
+	Name        string      `json:"name"`
+	ResourceRef ResourceRef `json:"resourceRef"`
+	FieldRef    []FieldRef  `json:"fieldRef"`
+}
+
+type Jinja2 struct {
```

Comment:
Shouldn't this be part of the Jinja expander rather than the composition?

Reply:
Jinja Expander has no CRD for itself. ExpanderVersion is not the right place. 
Iam expecting few common built-in expanders (core expanders) that are part of the system. 

Along those lines jinja2 is part of the built-in for now.

Reply:
so does that mean that we will also have a struct for the second expander that we are adding?

Reply:
thats correct. 
Added a comment to that effect: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/c0b57d8e17aa41097a1a892019b0b3371e958b97/experiments/compositions/composition/api/v1alpha1/composition_types.go#L79

Reply:
Seems ok for now but we need to think through the adding a new expander CUJ. I think the best way to do that it actually be uniform in how we add expanders. Toward that end I think we should have the expanders configuration CRD as another part of the expander package along with relevant roles, permissions etc.

Reply:
I see your intent. 
At the same-time having inline-expander config has simpler UX.

Wondering whats the best way. May be i can add a third expander that is properly offline.

Reply:
Adding a comment and a TODO to not have inline expanders long run